main.go

```diff
@@ -107,6 +113,10 @@ func init() {
 	flags.StringVar(&sentryDSN, "dsn", "", "Sentry DSN")
 	flags.BoolVarP(&verbose, "verbose", "v", false, "Verbose logging")
 
+	if env := os.Getenv("INTERFACE"); env != "" {
```

Comment:
```suggestion
	if env := os.Getenv("HOST"); env != "" {
```
(for consistency with flag)